docs/_src/api/api/document_store.md

```diff
@@ -22,21 +22,22 @@ Indexes documents for later queries.
 
 **Arguments**:
 
-- `documents`: a list of Python dictionaries or a list of Haystack Document objects.
                   For documents as dictionaries, the format is {"text": "<the-actual-text>"}.
                   Optionally: Include meta data via {"text": "<the-actual-text>",
                   "meta":{"name": "<some-document-name>, "author": "somebody", ...}}
                   It can be used for filtering and is accessible in the responses of the Finder.
-- `index`: Optional name of index where the documents shall be written to.
               If None, the DocumentStore's default index (self.index) will be used.
-- `batch_size`: Number of documents that are passed to bulk function at a time.
-- `duplicate_documents`: Handle duplicates document based on parameter options.
                             Parameter options : ( 'skip','overwrite','fail')
                             skip: Ignore the duplicates documents
                             overwrite: Update any existing documents with the same ID when adding documents.
                             fail: an error is raised if the document ID of the document being added already
                             exists.
 
+- `documents`: a list of Python dictionaries or a list of Haystack Document objects.
```

Comment:
Is this right? Seems like we cut the arg description here and having the rest still a few lines above?

Reply:
@tholor Good catch! There was a problem with the new pydoc-markdown version that we are using for the rendering of the files. I went back to pydoc-markdown==3.11.0. Now, it looks better.